.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
From the tsconfig file "outDir": "./dist", which means that we also need to ignore the compiled JS code.